components/pricings/type.ts

```diff
@@ -0,0 +1,25 @@
+export interface RegularPlan {
+  name: string;
+  displayedName: string;
+  mainPriceDisplay: string;
+  secondaryPriceDisplay: string;
+  isMostPopular: boolean;
+  features: any[][];
+}
+
+export interface StarterPlan {
+  name: string;
+  displayedName: string;
+  mainPriceDisplay: string;
+  isStarter: true;
+  features: any[][];
+}
+
+export interface CustomPlan {
+  name: string;
+  displayedName: string;
+  isCustom: true;
+  features: any[][];
+}
```

Comment:
Do the plans come from the backend in this format? 
Is there any reason why we don't have a type 'popular' | 'starter' | 'custom' instead of individual boolean values?

Reply:
so backend only send the paid plan. I need to hardcode the starter (free), and the custom.